api/src/test/java/org/pmiops/workbench/compliancetraining/ComplianceTrainingServiceTest.java

```diff
@@ -365,22 +277,9 @@ public void testSyncComplianceTrainingStatus_Absorb_DoesNothingIfUserHasntSigned
     assertModuleNotCompleted(DbAccessModuleName.CT_COMPLIANCE_TRAINING);
   }
 
-  @Test
-  public void testSyncComplianceTrainingStatus_Moodle_DoesNothingIfNoCoursesComplete()
```

Comment:
This is no longer possible since a user who has completed no Moodle courses will use Absorb instead.